packages/orbit-design-tokens/src/dictionary/transforms/attributes/index.ts

```diff
@@ -3,73 +3,63 @@ import { Attributes, Property } from "style-dictionary";
 
 import { errorTransform } from "../../utils/errorMessage";
 
+const NOV_STRUCTURE = ["namespace", "object", "variant", "subVariant"];
+
 /*
   The basic attribute transformer, adds serialized path info to the property.
-  For colors:
-  attributes: {
-    category: "palette",
-    name: "blue",
-    type: "light",
-    // default is not present
-  }
-
-  For other types:
   attributes: {
-    category: "base",
-    name: "space",
-    type: "XXS",
-    // default is not present
+    namespace: "foundation" (or "global", "variant"),
+    object: "palette" (or e.g. "border-radius"),
+    variant: "blue" (or e.g. "small"),
+    subVariant: "light" (nullish for most properties),
```

Comment:
Would this be more correct?
```suggestion
    subVariant: "light" (nullish for most variants),
```

Reply:
Yes it would! 🙂 Thanks for the suggestion.